eng/scripts/Get-AksSkus.ps1

```diff
@@ -0,0 +1,60 @@
+param(
```

Comment:
Is this script called anywhere? It seems like a remnant from doing some manual live testing.